src/integration/hubspotUpdatesController.ts

```diff
@@ -17,55 +17,60 @@ export class hubspotUpdatesController{
-            console.log(params);
-            // 1. Check if contact exists as user in SaaSquatch (match by email)
-            this.saasApiModel.getUsers(params)
-            .then( data =>{
-                //If it does not exist, create new user in SaaSquatch
-                if(data.count == 0){
-                    console.log("User does not exist in SaaSquatch");
-                    const createParticipantBody = {
-                            "email": participant.properties.email,
-                            "firstName": participant.properties.firstname,
-                            "lastName": participant.properties.lastname,
-                            "id": participant.properties.email,
-						    "accountId": participant.properties.email,
-            
-                    };
-                    this.saasApiModel.createParticipant(participant.properties.email, createParticipantBody);
-                }
-                // 3. TODO: If it does exist, get share link and other relevant data
-                else{
-                    console.log("SAAS USER "+data.users[0].email);
-                    console.log("user share links: "+data.users[0].shareLinks);
-                }
-                // 4. TODO: send referral link back to hubspot to add to contact
-            
-                }
-    
-            );
-        });
+        this.hubApiModel.getContact(contactObjectId,hub_id)
+            .then(data =>{
+                console.log("response");
+                console.log(data);
+                const participant = data;
+                params = `email:${encodeURIComponent(data.properties.email)}`;
+                console.log("PARAMS");
+                console.log(params);
+                // 1. Check if contact exists as user in SaaSquatch (match by email)
+                this.saasApiModel.getUsers(params)
+                    .then( data =>{
+                            //If it does not exist, create new user in SaaSquatch
+                            if(data.count == 0){
+                                console.log("User does not exist in SaaSquatch");
+                                const createParticipantBody = {
+                                    "email": participant.properties.email,
+                                    "firstName": participant.properties.firstname,
+                                    "lastName": participant.properties.lastname,
+                                    "id": participant.properties.email,
+                                    "accountId": participant.properties.email,
+
+                                };
+                                this.saasApiModel.createParticipant(participant.properties.email, createParticipantBody);
+                            }
+                            // 3. TODO: If it does exist, get share link and other relevant data
+                            else{
+                                console.log("SAAS USER "+data.users[0].email);
```

Comment:
here too? not sure if this was from backend people and if they need it.

Reply:
A lot of them are from the backend team, I removed some, some are still there. Remaining can be cleaned up in the refactoring ticket when we are able to have end to end functionality to test with.